source/retryable-writes/tests/insertOne-serverErrors.yml

```diff
@@ -262,6 +262,28 @@ tests:
                     - { _id: 1, x: 11 }
                     - { _id: 2, x: 22 }
                     - { _id: 3, x: 33 }
+    -
+        description: "InsertOne succeeds after ExceededTimeLimit"
+        failPoint:
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data:
+                failCommands: ["insert"]
+                writeConcernError:
+                    code: 262
+                    closeConnection: false
```

Comment:
The `closeConnection` option should sit parallel to the `failCommands` option directly under `data`.